### PR TITLE
easy question 2 in javascript

### PR DESCRIPTION
**Pull request by ridsashabbir**

### CODE REVIEW

File: Medium/question_1.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+class Dog {
+  bark() {
+    console.log("A dog can bark.");
+  }
+}
+
+function makeNoise(animal) {
+  // The makeNoise function expects an object with a "makeNoise" method
+  if (animal && typeof animal.makeNoise === "function") {
+    animal.makeNoise();
```
Comment (saadali18) [inline Medium/question_1.js:22]: `if (animal && typeof animal.bark === "function") animal.bark();`

**Review by saadali18 — CHANGES_REQUESTED** [review]

**Comment by saadali18** [comment]
PR looks good.
Few Suggestions: 

1. Use Appropriate name for your PR like **Added Solutions in JavaScript** as nobody from outside knows what easy question 2 is. 
2. Always add comments to your PR, to explain what is the purpose of your pull request. 
3. For code related part, avoid unnecessary comments. Adding comments in code is a good habit. But adding comments on obvious things is a not good approach. You can read **clean code** book to improve your coding style

File: Medium/question_1.js (hunk 1 above)
Comment (ridsashabbir) [reply Medium/question_1.js:22]: okay, let me correct this one, and thank you for the suggestions I'll work on them for sure.

Comment (saadali18) [reply Medium/question_1.js]: After you made changes, mark each comment resolved and request review again.
Screenshot 2023-11-09 at 8 58 28 AM

**Pull request merged**